plugins/woocommerce/src/StoreApi/Routes/V1/Agentic/CheckoutSessionsComplete.php

```
@@ -0,0 +1,430 @@
+
+		return $payment_data;
+	}
+
+	/**
+	 * Gets the chosen payment method (gateway) ID for CheckoutTrait.
+	 *
+	 * @param \WP_REST_Request $request Request object.
+	 * @return string
+	 * @throws RouteException If no payment gateway is available.
+	 */
+	private function get_request_payment_method_id( \WP_REST_Request $request ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)